src/figuresClasses.ts

```diff
@@ -1,11 +1,90 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
 
-export class Triangle implements Figure {}
+  color: string;
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a + b <= c || c + b <= a || a + c <= b) {
+      throw new Error(
+        'The sum of any two sides must be greater than the remaining side.',
+      );
+    }
+    this.color = color;
+    this.shape = 'triangle';
+    this.a = a;
+    this.b = b;
+    this.c = c;
```

Comment:
it is not necessary to specify it, when you specify parameters in the constructor of a class with the `public` keyword
```suggestion
```